homeassistant/components/overkiz/entity.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         if self.is_sub_device:
             # In case of sub entity, use the provided label as name
             self._attr_name = self.device.label
+        else:
+            self._attr_name = None
```

Comment:
This whole block can be removed. `self._attr_name = None` makes the entity get the name of the device. (check the DeviceInfo class later on in this file). The name of the deviceInfo is always `self.device.label`, so this if statement would do the same either way (in the if via direct naming, in the else via device naming). So you can remove this block and just set `_attr_name = None` at the top of the file (near _attr_has_entity_name)

Reply:
I am checking the rest of the code, and I am doubting that setting self._attr_name to None everywhere is the way to go. Currently because this integration does not use translation keys for the entity naming, it gets overwritten, but not sure what happens if you have translation keys there.

Reply:
First try to address this in 2c8c4873092286fcc044d317348c974b876ea436. Regarding translation, that will be added in a follow up PR.